SU-SWS/saa_victoria_subtheme
# Adapt 551  table label

# READY FOR REVIEW

# Summary
- Add a mobile-only small text label above tables

# Review By (Date)
- 8/7

# Urgency
- low

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. Check mobile for a label like this:
With updated text now:
![image](https://user-images.githubusercontent.com/12898896/89594162-61ca2180-d80e-11ea-87af-698a6cae4083.png)



# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket  https://stanfordits.atlassian.net/browse/ADAPT-551
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
